File: DepthChartAPI/Models/Player.cs
using System;

namespace DepthChartAPI.Models;

public class Player
{
    public Guid Id { get; set; } = Guid.NewGuid();
    public string FirstName { get; set; } = string.Empty;
    public string LastName { get; set; } = string.Empty;
    public int Jersey { get; set; }
    public string Position { get; set; } = string.Empty;
    public int Order { get; set; }
    public string Status { get; set; } = string.Empty;
}

File: frontend/src/types/player.ts
export interface Player {
  id: string;
  firstName: string;
  lastName: string;
  jersey: number;
  position: string;
  order: number;
  status: string;
}

export type Phase = 'offense' | 'defense' | 'special-teams';

export interface PositionGroup {
  name: string;
  positions: string[];
  phase: Phase;
}
